ml-completion/src/test/resources/META-INF/plugin.xml

```diff
@@ -0,0 +1,5 @@
+<idea-plugin xmlns:xi="http://www.w3.org/2001/XInclude" allow-bundled-update="true">
+    <id>org.rust.ml</id>
+    <xi:include href="/META-INF/rust-core.xml" xpointer="xpointer(/idea-plugin/*)"/>
+    <depends optional="true" config-file="ml-completion-only.xml">com.intellij.stats.completion</depends>
```

Comment:
plugin id is not updated to `com.intellij.completion.ml.ranking`, as a result, test on 2020.3 fail